controlplane/src/core/repositories/analytics/util.ts

```diff
@@ -189,10 +191,22 @@ export function buildCoercedFilterSqlStatement(
       // https://clickhouse.com/docs/en/interfaces/cli#cli-queries-with-parameters
       // https://clickhouse.com/docs/en/sql-reference/data-types
 
-      if (column.type === 'number') {
-        sql = `${dbField} ${operatorSql} {${id}:Float64}`;
-      } else if (column.type === 'string') {
-        sql = `${dbField} ${operatorSql} {${id}:String}`;
+      switch (column.type) {
+        case 'number': {
+          sql = `${dbField} ${operatorSql} {${id}:Float64}`;
+          break;
+        }
+        case 'string': {
+          sql = `${dbField} ${operatorSql} {${id}:String}`;
+          break;
```

Comment:
This is not necessary but I also don't want to remove it because it provides support for boolean values.